en/manual/xenko-for-unity-developers/index.md

```diff
@@ -579,7 +615,7 @@ public Quaternion SpawnRotation;
 public override void Start()
 {
     // Initialization of the script.
-    var car = CarPrefab.Instantiate();
+    List<Entity> car = CarPrefab.Instantiate();
     SceneSystem.SceneInstance.RootScene.Entities.AddRange(car);
     car.First().Transform.Position = SpawnPosition;
     car.First().Transform.Rotation = SpawnRotation;
```

Comment:
While we are at it, let's make that code sample better:

```csharp
     List<Entity> car = CarPrefab.Instantiate();
    SceneSystem.SceneInstance.RootScene.Entities.AddRange(car);
    car[0].Transform.Position = SpawnPosition;
    car[0].Transform.Rotation = SpawnRotation;
```

Reply:
Sorry for the long delay, thought your changes were automatically added. Added a new commit to fix up the code sample.